src/plugins/vuetify.ts

```diff
@@ -17,6 +18,9 @@ export default createVuetify({
       mdi,
     },
   },
-  components,
+  components: {
+    ...components,
+    VDataTable
+  },
```

Comment:
add this, otherwise `v-data-table` component cannot be founded
https://stackoverflow.com/questions/65669486/v-data-table-component-not-found

Reply:
remove that one and with the updated Vuetify version, it works

Reply:
Great news! I think you can resolve this conversation then :)